src/cov.jl

```diff
@@ -29,6 +29,21 @@ scattermat_zm(x::DenseMatrix, wv::WeightVec, vardim::Int) =
     _symmetrize!(Base.unscaled_covzm(x, _scalevars(x, values(wv), vardim), vardim))
 
 if VERSION < v"0.5.0-dev+679"
+"""
+    scattermat(x, [wv,] [mean=nothing,] [vardim=1]) -> Matrix
+
+Compute the scatter matrix, optionally specifying a weighting vector.
```

Comment:
Could you explain what is a "scatter matrix" in a few words?

Regarding the recommended ways of presenting keyword arguments, see point 4 at http://docs.julialang.org/en/latest/manual/documentation/.